src/xhr.ts

```diff
@@ -41,6 +41,39 @@ export class JQueryRequestor extends Requestor {
   }
 }
 
+/**
+ * Uses fetch API to make Ajax requests
+ */
+export class FetchRequestor extends Requestor {
+  xhr<T>(settings: JQueryAjaxSettings): Promise<T> {
+    const url: string = settings.url || '';
+    let requestInit: RequestInit = {};
+    requestInit.method = settings.method;
+    
+    if (settings.data) {
+      requestInit.body = <string>settings.data;
+    }
+    
+    //Calculate 'Accept' header value
+    let headers = settings.headers || {};
+    if (settings.dataType && settings.dataType.toLowerCase() === "json"){
+      headers['Accept'] = 'application/json, text/javascript, */*; q=0.01';
```

Comment:
Do this after you mixin other header key value pairs. Otherwise the developers `Accept` header will overwrite your `Accept` header.

Reply:
Probably I'm missing something obvious here but isn't actually the other way around? If the caller sets an `Accept` header, it will be overwritten.

Reply:
Nit: I think you should mix-in your header values _after_ developer provided values. That way your values win.